src/main/java/seedu/savenus/logic/commands/DislikeCommand.java

```diff
@@ -0,0 +1,51 @@
+package seedu.savenus.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Set;
+
+import seedu.savenus.logic.commands.exceptions.CommandException;
+import seedu.savenus.model.Model;
+import seedu.savenus.model.food.Category;
+import seedu.savenus.model.food.Location;
+import seedu.savenus.model.tag.Tag;
+
+
+/**
+ * Adds the user's disliked recommendations to the $aveNUS recommendation system.
+ */
+public class DislikeCommand extends PreferenceCommand {
+
+    public static final String COMMAND_WORD = "dislike";
+
+    private final Set<Category> categoryList;
+    private final Set<Tag> tagList;
+    private final Set<Location> locationList;
```

Comment:
These fields are only used in the equals method which is the same for both the Like and Dislike Command. Would there be future uses of these fields? Otherwise I would recommend deleting these fields and moving the equals method into the Preference Command.

Reply:
Good idea 👍